app/helpers/products_helper.rb

```diff
@@ -1,4 +1,73 @@
 module ProductsHelper
+
+  class RenderReadmeMarkdown < Redcarpet::Render::HTML
```

Comment:
This should be moved to a separate class. Probably under another directory under app/. Not sure which yet. Until then could move to app/models/ and call the class ProductReadmeMarkdownRenderer. FWIW I don't know why the other renderer is called `RenderManifestMarkdown` which is a verb, not a noun.